changelogs/unreleased/7531-race-condition-in-exporter-file-upload.yml

```diff
@@ -0,0 +1,11 @@
+description: >
+  The upload_file endpoint will now silently ignore attempts to upload a file if a file with the same
+  hash was previously uploaded.
+issue-nr: 7531
+change-type: minor
+destination-branches: [master, iso7]
+sections:
+  minor-improvement: "{{description}}"
```

Comment:
To me this change is just a bugfix. I don't really see why it needs a minor-improvement entry.